src/universal.css

```diff
@@ -3,20 +3,23 @@
   list-style: none;
 }
 
-html, body {
+html,
+body {
   overflow: hidden;
 }
 
 html {
   background-color: var(--default-background-color);
 }
 
-body, input {
+body,
+input {
   font-family: 'NB International Pro';
 }
 
-code {
-  word-break: break-word
+code,
+blockquote {
```

Comment:
This is the actual specific intention of the commit. The rest is just general nits